.github/workflows/PR.yml

```diff
@@ -31,6 +31,6 @@ jobs:
         run: |
           poetry run ni-python-styleguide lint
       - name: Generate ni-daqmx files
-        run: poetry run python -m codegen --dest generated/nidaqmx
```

Comment:
It didn't report the error because `click` ignores the command callback's return value by default. I want the codegen script to display a stack trace, so I removed the try/except block so that the command propagates exceptions. If I didn't want it to display a stack trace, I would raise `click.ClickException("some error message")`.

It couldn't find the template file because `script_info.py` uses Windows path separators and the GitHub workflow runs the script on Linux. I used `pathlib.PureWindowsPath` to convert the path separator.